response_operations_ui/__init__.py

```diff
@@ -105,5 +132,7 @@ def user_loader(user_id):
     Session(app)
 
     setup_blueprints(app)
+    csrf.exempt(reporting_unit_bp)
+    csrf.exempt(messages_bp)
```

Comment:
why are these pages exempt?

Reply:
Let's discuss offline how to smoke test without these exemptions.